jwt/validate.go

```diff
@@ -175,6 +175,34 @@ func (err *missingRequiredClaimError) Is(target error) bool {
 func (err *missingRequiredClaimError) isValidationError() {}
 func (*missingRequiredClaimError) Unwrap() error          { return nil }
 
+type invalidAudienceError struct {
```

Comment:
(Note, this is for me, you don't have to do anything @ste93cry )
This should just be generated so it's easier to add errors in the future